### ISSUE TITLE
error when no connection

### ISSUE THREAD

**Issue by danielmedina** [giggio/speedtest#3]
Hi.

First of all, great application!

I got an error when executing cronjob when I got disconnected from my link. Then I noticed that  the exection wasn't writen on the csv file. I believe you should write when there is no connection too because it will help the user to discover when his internet is unstable. The real problem is that I don't know  a thing in Rust and can't help you with that.

Btw, this is the error I got using in a Mac:

```
Speedtest executable exited with an error. Output:

Errors:
[2021-03-23 17:10:02.399] [error] Trying to get interface information on non-initialized socket.
[2021-03-23 17:10:02.429] [error] Configuration - Couldn't resolve host name (HostNotFoundException)
[2021-03-23 17:10:02.430] [error] Configuration - Cannot retrieve configuration document (0)
[2021-03-23 17:10:02.430] [error] ConfigurationError - Could not retrieve or read configuration (Configuration)
[2021-03-23 17:10:02.430] [error] ConfigurationError - Could not retrieve or read configuration (Configuration)
{"type":"log","timestamp":"2021-03-23T17:10:02Z","message":"Configuration - Could not retrieve or read configuration (ConfigurationError)","level":"error"}
```

**Comment by giggio** [comment]
Maybe we could send an alert to the user's email when this happens? Because writing zeros on the file would actually show incorrect information.

**Comment by danielmedina** [comment]
I don't know.... I was trying to use it to check when my link was down to
complain about it with my provider. I think if it was informed in csv
file,, I could done that...

Em sáb, 3 de abr de 2021 11:16, Giovanni Bassi ***@***.***>
escreveu:

**Comment by giggio** [comment]
You now have the option to send an email if you can't get a connection, and it will log it to stdout if there is a problem. Fixed on abf23b43283ddea8a577fa301706bf78b4b0eea2.
